docs/platform/howto/pause-from-cli.md

```diff
@@ -1,9 +1,16 @@
 ---
 title: Pause or terminate your service
+sidebar_label: Pause or terminate service
 ---
 
 Cloud services can be created, paused, or destroyed as needed, to optimize costs. For example, you can power the service off temporarily to save credits.
 
+:::important
+Static IP addresses are not disassociated/deleted when a service is powered off or
+deleted. They continue to generate the usual costs. To remove static IP addresses, see
+[Manage static IP addresses](/docs/platform/howto/static-ip-addresses).
+:::
+
```

Comment:
Let's make this a reusable component 🙌  

1. Create a file in `static/includes` that contains your note.
2. Import it in both files: `Import StaticIp from "@site/static/includes/yoursnippet.md";` 
3. Use where appropriate with `<StaticIp/>`